src/pipeline/extract_employee_data.py

```diff
@@ -0,0 +1,126 @@
+from os import curdir
+import psycopg2 as pg2
+import json 
+import xmltodict
+
+def connect():
+    return pg2.connect(
+                host = "localhost",
+                user = "postgres",
+                password = "sql-admin",
+                port = "5432",
+                database = "employee_timesheet_db"
+            )
+
+def create_raw_employee_data_schema():
+    conn = connect()
+    cursor = conn.cursor()
+
+    with open('../../schema/create_raw_employee_data.sql') as create_file:
+        create_query = "".join(create_file.readlines())
+        cursor.execute(create_query)
+        conn.commit()
+    cursor.close()
+    conn.close()
+
+def delete_existing_records(file_path):
+    conn = connect()
+    cursor = conn.cursor()
+    with open(file_path) as delete_file:
+        delete_query = "".join(delete_file.readlines())
+        cursor.execute(delete_query)
+        conn.commit()
+    cursor.close()
+    conn.close()
+
+def extract_employee_data_csv(file_path, flag):
+    conn = connect()
+    cursor = conn.cursor()
+
+    with open(file_path, 'r') as file:
+        if flag:
+            delete_existing_records('../sql/procedure/delete_raw_employee_data.sql')
+
+        next(file)
+        for line in file:
+            row = line.strip().split(',')
+
+            with open('../sql/procedure/insert_raw_employee_data.sql') as insert_file:
+                insert_query = "".join(insert_file.readlines())
+                cursor.execute(insert_query, row)
+                conn.commit()
+    cursor.close()
+    conn.close()
```

Comment:
Same here. Extraction is the same process. the only difference is the source and sink. So, maybe we can generalize this?